defaults/main.yml

```diff
@@ -105,7 +105,7 @@ bbb_dialin_provider_username: "provider-account"
 bbb_dialin_provider_password: "provider-password"
 bbb_dialin_provider_extension: "6135551234"
 bbb_dialin_default_number: "613-555-1234"
-bbb_dialin_default_play_and_get_digits: "5 5 3 7000 #"
+bbb_dialin_default_play_and_get_digits: "5 5 3 7000 # conference/conf-pin.wav ivr/ivr-that_was_an_invalid_entry.wav pin \\d+"
```

Comment:
Hi @wilkis3, is the double-backslash here needed for escaping? If so, i would suggest to put this in readme as well.